core/__tests__/tasks/profileProperty/importProfileProperties.ts

```diff
@@ -111,6 +111,35 @@ describe("tasks/profileProperty:importProfileProperties", () => {
       await profile.destroy();
     });
 
+    test("will set value to null for profile properties that no longer exist", async () => {
+      const spy = jest
+        .spyOn(testPluginConnection.methods, "profileProperties")
+        .mockImplementation(() => undefined);
```

Comment:
This test works because `canImport` will still be true?  If that's the reason, perhaps that should be added to the test somehow.

Reply:
No, `canImport` only applies when importing synchronously (e.g. import and export button). In this case, this works because the `importProfileProperties` task now sets `rawValue: null`. This task is never enqueued if the plugin doesn't have an `importProfileProperties` method (as tested by `__tests__/tasks/profileProperty/enqueue: if there is no import method, it will just mark properties as ready`)